tensorflow/docs_src/programmers_guide/using_tpu.md

```diff
@@ -129,10 +129,10 @@ my_tpu_estimator = tf.contrib.tpu.TPUEstimator(
 Typically the `FLAGS` would be set by command line arguments. To switch from
 training locally to training on a cloud TPU you would need to:
 
-  1) Set `FLAGS.use_tpu` to `True`
-  1) Set `FLAGS.tpu_name` so the
+* Set `FLAGS.use_tpu` to `True`
+* Set `FLAGS.tpu_name` so the
      `tf.contrib.cluster_resolver.TPUClusterResolver` can find it
```

Comment:
Fix the indent before `tf.contrib.cluster_resolver.TPUClusterResolver`

Reply:
Thanks for comment, fixed the indent.